src/components/pages/BugReportPage.js

```diff
@@ -2,13 +2,18 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { withStyles } from '@material-ui/styles';
 import { pageContainer } from './common/styles';
+import PageTitle from './common/PageTitle';
 
 export const styles = {
   pageContainer,
 };
 
 function BugReportPage({ classes }) {
-  return <div className={classes.pageContainer} />;
+  return (
+    <div className={classes.pageContainer}>
+      <PageTitle name="Report a Bug" />
```

Comment:
I think we can use a better prop name than `name` - maybe something like `title`?